### DISCUSSION TITLE
move workspace is crashing i3

### DISCUSSION THREAD

**Discussion by sfischme** [i3/i3#5581] [Q&A]
This little script is crashing i3wm (4.22) after unplugging a second monitor, having the screens auto-adjusted (autorandr), and restarting i3 (```i3-msg restart```):
```
i3-msg -t get_workspaces | jql '|={"output"="xinerama-1"}' | sed -n 's/.*name.: "\(.*\)",/\1/p' > /tmp/list-1

for i in $(cat /tmp/list-1); do
    i3-msg [workspace=$i] move workspace to output xinerama-0
done
```
Sometimes it's working, other times it's consistently crashing i3 showing me the info box to restart. If it crashes, subsequent executions will also crash i3.

Am I am doing something wrong there?

PS: The point of the script is to move all workspaces to the single screen after unplugging the second screen.

**Comment by orestisfl** [comment]
Please open a [new](https://github.com/i3/i3/issues/new/choose) bug report and include logs and a backtrace as well.

To allow gdb to acquire a backtrace, you might need to run `echo 0 | sudo tee /proc/sys/kernel/yama/ptrace_scope` beforehand.
